dist/js/components/Booking.js

```diff
@@ -31,7 +171,86 @@ class Booking{
     thisBooking.hoursAmount = new AmountWidget(thisBooking.dom.hoursAmount);
     thisBooking.datePicker = new DatePicker(thisBooking.dom.datePicker);
     thisBooking.hourPicker = new HourPicker(thisBooking.dom.hourPicker);
-  }  
+
+    thisBooking.dom.wrapper.addEventListener('updated', function(){
+      thisBooking.removeTables();
+      thisBooking.updateDOM();
+    }); 
+    
+    thisBooking.dom.floorPlan.addEventListener('click', function (event){
+      thisBooking.initTables(event);
+    });
+
+    thisBooking.dom.bookingButton.addEventListener('click', function (event){
+      event.preventDefault();
+      thisBooking.sendBooking();
+    });    
+  }
+
+  removeTables(){
+    const thisBooking = this;
+ 
+    for (let table of thisBooking.dom.tables) {
+      table.classList.remove(classNames.booking.tableSelected);
+    }
+  }
+
+  initTables(event){
+    const thisBooking = this;
+    const tableId = event.target.getAttribute('data-table');
+    const isBooked = event.target.classList.contains(classNames.booking.tableBooked);
+    const isSelected = event.target.classList.contains(classNames.booking.tableSelected);
+
+    if(tableId) {
+      if(isBooked) {
+        alert('This table is not available.');
+      } else if(isSelected) {
+        event.target.classList.remove(classNames.booking.tableSelected);
+        thisBooking.selected = {};
+      } else if(!isSelected) {
+        thisBooking.removeTables();
+        event.target.classList.add(classNames.booking.tableSelected);
+        thisBooking.selected = tableId;
+      }
+    }
+  }
+  
+  sendBooking() {
+    const thisBooking = this;
+    const url = settings.db.url + '/' + settings.db.bookings;
+    const payload = {
+      date: thisBooking.datePicker.value,
+      hour: thisBooking.hourPicker.value,
+      table: parseInt(thisBooking.selected),
+      duration: parseInt(thisBooking.hoursAmount.value),
+      ppl: parseInt(thisBooking.peopleAmount.value),
+      starters: [],
+      phone: thisBooking.dom.phone.value,
+      address: thisBooking.dom.address.value,
+    };
+
+    for (let starter of thisBooking.dom.starters){
+      if (starter.checked){
+        payload.starters.push(starter.value);
+      }
+    }
+
+    const options = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(payload)
+    };
+    
+    fetch(url, options)
+      .then(function(response){
+        return response.json();
+      }).then(function(parsedResponse){
+        console.log('parsedResponse', parsedResponse);
+        thisBooking.makeBooked(payload.date, payload.hour, payload.duration, payload.table);
+    });
```

Comment:
Hej! Możesz tutaj użyć danych, które dostajesz w odpowiedzi z POST request. Na końcu powinnaś wywołać również metodę `updateDOM`, żeby zaktualizować wyświetlane elementy:
```suggestion
      }).then(function(data){
        console.log('data', data);
        thisBooking.makeBooked(data.date, data.hour, data.duration, data.table);
    });
    thisBooking.updateDOM();
```